lib/ansible/modules/source_control/perforce/helix_configurable.py

```diff
@@ -0,0 +1,220 @@
+        required: true
+        type: str
+    value:
+        description:
+            - The value of named configurable
+        required: true
+        type: str
+    p4port:
+        description:
+            - The hostname/ip and port of the server (perforce:1666)
+        required: true
+        type: str
+    p4user:
+        description:
+            - A user with super user access
+        required: true
+        type: str
+    p4passwd:
+        description:
+            - The super user password
+        required: true
+        type: str
+    serverid:
+        default: any
+        description:
+            - The server ID of the helix server
+        required: false
+        type: str
+
+author:
+    - Asif Shaikh (@ripclawffb)
+'''
+
+EXAMPLES = '''
+# Set auth.id configurable for any server
+- name: Set auth.id
+  helix_configurable:
+    state: present
+    name: auth.id
+    value: master.1
+    p4port: '1666'
+    p4user: bruno
+    p4passwd: ''
+
+# Unset auth.id configurable for specific server
+- name: Unset auth.id
+  helix_configurable:
+    state: absent
+    name: auth.id
+    value: master.1
+    serverid: master.1
+    p4port: '1666'
+    p4user: bruno
+    p4passwd: ''
+'''
+
+RETURN = r''' # '''
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.six import string_types
+
+try:
+    from P4 import P4, P4Exception
+    HAS_P4 = True
+except ImportError:
+    HAS_P4 = False
+
+
+def helix_connect(p4_user, p4_port, p4_password, script_name):
+    """
+    Pass this function a user, p4port, password and client to connect to
+    the server
+    """
+    try:
+        p4 = P4()
+        p4.prog = script_name
+        p4.port = p4_port
+        p4.user = p4_user
+        p4.password = p4_password
+        p4.connect()
+        p4.run_login()
+        if p4.connected() is not True:
+            raise RuntimeError
+        return p4
+    except Exception as e:
+        return "There was a problem in connecting to Helix: {0}".format(e)
+
+
+def helix_disconnect(connection):
+    """
+    Pass this function a connection object to disconnect the Helix Core
+    session
+    """
+
+    connection.disconnect()
+
+
+def run_module():
+    # define available arguments/parameters a user can pass to the module
+    module_args = dict(
+        state=dict(type='str', default='present', choices=['present', 'absent']),
+        name=dict(type='str', required=True),
+        value=dict(type='str', required=True),
+        p4port=dict(type='str', required=True),
+        p4user=dict(type='str', required=True),
+        p4passwd=dict(type='str', required=True, no_log=True),
+        serverid=dict(type='str', default='any')
+    )
+
+    result = dict(
+        changed=False,
+    )
+
+    module = AnsibleModule(
+        argument_spec=module_args,
+        supports_check_mode=True
+    )
+
+    if not HAS_P4:
+        module.fail_json(msg="Missing required lib 'p4python'")
```

Comment:
Use [missing_required_lib](https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/basic.py#L562) to generate the message.

Reply:
Updated to use missing_required_lib.

Reply:
Updated to use missing_required_lib